roxie/ccd/ccdfile.cpp

```diff
@@ -314,8 +314,8 @@ class CRoxieLazyFileIO : implements ILazyFileIO, implements IDelayedFile, public
                 }
             }
             lastAccess = msTick();
-            atomic_inc(&numFilesOpen[remote]);
-            if ((unsigned) atomic_read(&numFilesOpen[remote]) > maxFilesOpen[remote])
+            ++numFilesOpen[remote];
```

Comment:
Minor, but this might be clearer and more efficient if the two lines were combined